tests/e2e/default/enketo/enketo-widgets.wdio-spec.js

```diff
@@ -0,0 +1,155 @@
+const fs = require('fs');
+const placeFactory = require('@factories/cht/contacts/place');
+const userFactory = require('@factories/cht/users/users');
+const utils = require('@utils');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const enketoWidgetsPage = require('@page-objects/default/enketo/enketo-widgets.wdio.page');
+const genericForm = require('@page-objects/default/enketo/generic-form.wdio.page');
+const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
+const contactPage = require('@page-objects/default/contacts/contacts.wdio.page');
+
+
+describe('Enketo Widgets', () => {
+  const districtHospital = placeFactory.place().build({ _id: 'dist1', type: 'district_hospital' });
+  const offlineUser = userFactory.build({
+    place: districtHospital._id,
+    roles: ['chw'],
+    contact: {_id: '987 654 321', name: 'Ben', phone: '+50689999999'}
```

Comment:
Can you leave an inline comment here noting that the `name` value is deliberately `< 4` characters so that it violates the inputs constraint in the form?